base/gfspconfig/config.go

```diff
@@ -222,3 +223,7 @@ type MetadataConfig struct {
 	IsMasterDB                 bool
 	BsDBSwitchCheckIntervalSec int64
 }
+
+type ManagerConfig struct {
```

Comment:
load should be a must, why need config?

Reply:
If there is a bug in the load, it may not be able to start; this may not be acceptable, and the configuration is more flexible.